start-client/src/components/Application.js

```diff
@@ -71,10 +71,15 @@ export default function Application() {
       get(dependencies, 'list')
     ).catch(() => {
       toast.error(`Could not connect to server. Please check your network.`)
-      setGenerating(false)
     })
     setGenerating(false)
-    FileSaver.saveAs(project, `${get(values, 'meta.artifact')}.zip`)
+    if (project) {
+      if (project.type == "text/plain") {
+        project.text().then(text => toast.info(text))
```

Comment:
There is already an error message (generic), this info message is useless.

Reply:
Currently, yes. But it provides a useful option if the backend wanted to send a more specific message. It's not hurting anyone, and it might help.

Reply:
`getProject` return a promise (without data/message in case of rejection).
`project.type` will never be true.

Reply:
Sure. At least not with the current implementation of the backend. I was just suggesting an enhancement. Feel free to take it out if you feel strongly about it.

Reply:
Thanks Dave. I pushed a fix on the master (no ZIP creation in case of error).
https://github.com/spring-io/start.spring.io/commit/c425ac3a3ea9607c5bf94933576e0bf67e4badbb